platforms/linux/packaging/post-install-script.sh

```diff
@@ -2,7 +2,17 @@
 # Copyright (c) 2024 Husarnet sp. z o.o.
 # Authors: listed in project_root/README.md
 # License: specified in project_root/LICENSE.txt
-
+set -euo pipefail
+if ! grep -q "^husarnet:" /etc/group; then
+    groupadd husarnet
+fi
+if [ ! -d "/var/lib/husarnet" ]; then
+  mkdir -p /var/lib/husarnet
+fi
+chgrp husarnet /var/lib/husarnet -R
+chmod g+rwx /var/lib/husarnet -R
```

Comment:
This will `+x` also the files inside, which is undesirable. I believe only robust option for this is to use something like `find` and two calls - one for directories and one for files.